float stbd_angle
float port_angle
float aft_power
float stbd_power
float port_power
